Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -825,8 +826,10 @@ public static void HandleWeapons(Session session, params string[] parameters)
             {
                 foreach (uint weenieId in weaponsTest)
                 {
-                    WorldObject loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
+                    AceObject lootAceObject = (AceObject)DatabaseManager.World.GetAceObjectByWeenie(weenieId).Clone(GuidManager.NewItemGuid().Full);
+                    WorldObject loot = new GenericObject(lootAceObject);
```

Comment:
Ok, I will work on making the other method work and refactor my code to use that.